airflow/www/static/js/trigger.js

```diff
@@ -17,13 +17,14 @@
  * under the License.
  */
 
-/* global document, CodeMirror */
+/* global document, CodeMirror, window */
 
 const textArea = document.getElementById('json');
+const height = window.innerHeight * 0.6;
```

Comment:
```suggestion
const height = window.innerHeight - 450;
```
Window height minus headers and footers.

Reply:
And we should probably do a minHeight too like `height > 300 ? height : 300`

Reply:
Perhaps something like the below?
> const minHeight = 300;
const maxHeight = window.innerHeight - 450;
const height = maxHeight > minHeight ? maxHeight : minHeight;

Reply:
exactly

Reply:
Cool, I've added to the PR.